AuthenticatorBridgeKit/Tests/TestHelpers/Fixtures/AuthenticatorBridgeItemDataModel+Fixtures.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+@testable import AuthenticatorBridgeKit
+
+extension AuthenticatorBridgeItemDataModel {
+    static func fixture(
+        favorite: Bool = true,
+        id: String = UUID().uuidString,
+        name: String = "Name",
+        totpKey: String? = "TOTP Key",
+        username: String? = "Username"
+    ) -> AuthenticatorBridgeItemDataModel {
```

Comment:
⛏️ On fixtures I think it's better to rely on default values and then customize as needed or provide a particular `fixtureWhatever` func for customization. This helps when using this amongst different tests where it's more intuitive to know the value of a default parameter and the expect behavior of such. What do you think?
```suggestion
    static func fixture(
        favorite: Bool = false,
        id: String = UUID().uuidString,
        name: String = "Name",
        totpKey: String? = nil,
        username: String? = nil
    ) -> AuthenticatorBridgeItemDataModel {
```

Reply:
That sounds reasonable to me. 👍 I'll adjust this over in the next commit.